scripts/local-dev.sh

```diff
@@ -141,56 +146,44 @@ check_prerequisites() {
     echo -e "${GREEN}✅ Prerequisites check passed${NC}"
 }
 
+# Helper function to create port-forward command
+create_port_forward() {
+    local resource_name="$1"
+    local local_port="$2"
+    local remote_port="$3"
+    local namespace="$4"
+    local description="$5"
+    local emoji="$6"
+
+    # Check if resource name is found
+    if [ -z "$resource_name" ]; then
+        echo -e "${YELLOW}⚠️  $description service NOT found in $namespace namespace. Exiting...${NC}"
+        exit 1
+    fi
+    
+    # Create port-forward
+    oc port-forward "$resource_name" "$local_port:$remote_port" -n "$namespace" >/dev/null 2>&1 &
+    echo -e "${GREEN}✅ Found $description: $emoji (resource: $resource_name, namespace: $namespace) available at: http://localhost:$local_port${NC}"
+}
+
 # Function to find and start port forwards
 start_port_forwards() {
     echo -e "${BLUE}🔍 Finding pods and starting port-forwards...${NC}"
-    
-    # Find Thanos pod
-    THANOS_POD=$(oc get pods -n "$PROMETHEUS_NAMESPACE" -o name | grep thanos-querier | head -1 | cut -d'/' -f2 || echo "")
-    if [ -z "$THANOS_POD" ]; then
-        THANOS_POD=$(oc get pods -n "$PROMETHEUS_NAMESPACE" -o name | grep prometheus | head -1 | cut -d'/' -f2 || echo "")
-    fi
-    
-    if [ -n "$THANOS_POD" ]; then
-        echo -e "${GREEN}✅ Found Thanos pod: $THANOS_POD${NC}"
-        oc port-forward pod/"$THANOS_POD" "$THANOS_PORT:9090" -n "$PROMETHEUS_NAMESPACE" >/dev/null 2>&1 &
-        echo -e "${GREEN}   📊 Thanos available at: http://localhost:$THANOS_PORT${NC}"
-    else
-        echo -e "${RED}❌ No Thanos/Prometheus pod found${NC}"
-        exit 1
-    fi
+
```

Comment:
This is the ONLY place I'm using `pod` for port-forwarding as service didn't work here (_didn't spend too much time investigating it either_) but for all others I've now switched to using `service`.